src/packagers/JSConcatPackager.js

```diff
@@ -72,6 +72,9 @@ class JSConcatPackager extends Packager {
 
       // If this was an ES6 export all (e.g. export * from 'foo'), resolve to the original exports.
       if (dep.isExportAll) {
+        asset.cacheData.exports[`$${asset.id}$exports`] = `$${mod.id}$exports`;
```

Comment:
Is this true? What if you re-exported multiple modules?

```javascript
export * from './foo';
export * from './bar';
```

Then `$id$exports` should be the combination of both modules...

Reply:
Good catch, it's a tricky case, I'll prolly fix it by fixing the commonjs import